packages/ui-components/src/index.ts

```diff
@@ -0,0 +1,4 @@
+export {default as LoadingAnimation} from './loading-animation/LoadingAnimation'
```

Comment:
what do you think if the files inside the folders we called them inde.js? so we would avoid repetition

Reply:
Sincerely I'm not a big fan of index.js everywhere: we didn't use that approach in `microlc` because, when you search a component using the IDE, is more developer-friendly to have `component-name.tsx` than a infinite list of `index.ts`